js-intro.js

```diff
@@ -35,27 +62,76 @@ var myNumbers2 = [8, -7, 0, 6, 2]
 
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
-
+// creating a function called noVowels
+// create a newArr w/ vowels ["a","e","i","o","u"]
+noVowels = (thing) => {
+  let vowels = ["a","e","i","o","u"]
+  // using .split to get string into individual ""'s, filter the string for just vowels
+  return thing.split("").filter(function(el) {
+    //brings back the vowels and string, and subtract out the vowels from joined string
+    return vowels.indexOf(el.toLowerCase()) == -1;
+  }).join("");
+}
+console.log(noVowels(stringWithVowels1));
+console.log(noVowels(stringWithVowels2));
 
 
 
 
 // --------------------5) Expand the logic from #4 to inform your user if the variable passed into the vowel removal function is not a string.
 // Copy and paste or rewrite the function. Use the test variables provided below.
 
-var notAString1 = true
-var notAString2 = 42
-
+// copy code of #4 from above
+
+//this code is commented out because #4 and #5 will not run together but will both work and run seperatly. Not sure why
+
+// var notAString1 = true
+// var notAString2 = 42
+// var boo = typeof (true)
+// var num = typeof (1)
+// var str = typeof ("string")
+// //THE START OF COPIED CODE//////////////////////////
+// function noVowels(thing) {
+//   let vowels = ["a","e","i","o","u"]
+//   // condition for if !== string is typed using typeOf
+//   if (thing === boo || num)
+//     return `${thing}, is not a string.`
+//   else if (thing === str){
+//   // using .split to get string into individual ""'s, filter the string for just vowels
+//     return thing.split("").filter(function(el) {
+//       //brings back the vowels and string, and subtract out the vowels from joined string
+//       return vowels.indexOf(el.toLowerCase()) == -1;
+//     }).join("");
+//   }
+// }
+// // THE END OF COPIED CODE///////////////////////////
+// console.log(noVowels(notAString1));
+// console.log(noVowels(notAString2));
 
```

Comment:
noVowels = (thing) => {
  let vowels = ["a","e","i","o","u"]
  // using .split to get string into individual ""'s, filter the string for just vowels
  if(typeof thing === "string"){
    return thing.split("").filter(function(el) {
      //brings back the vowels and string, and subtract out the vowels from joined string
      return vowels.indexOf(el.toLowerCase()) == -1;
    }).join("")
  }
  return "Please enter a string."
}
I modified your code to use typeof for this example.